packages/edit-navigation/README.md

```diff
@@ -20,6 +20,103 @@ import blockEditorSettings from './block-editor-settings';
+| `post_title`              |                      Not mapped.                       |                   A "no title" label if menu item represents a post that lacks a title.                   |
+| `target`                  |    `opensInNewTab`<sup>[2](#menu_item_target)</sup>    |                       The target attribute of the link element for this menu item.                        |
+| `title`                   |                        `label`                         |                                       The title of this menu item.                                        |
+| `type`                    |                         `kind`                         |             The family of objects originally represented, such as 'post_type' or 'taxonomy'.              |
+| `type_label`              |                      Not mapped.                       |                        The singular label used to describe this type of menu item.                        |
+| `url`                     |                         `url`                          |                                  The URL to which this menu item points.                                  |
+| `xfn`                     |                         `rel`                          |                       The XFN relationship expressed in the link of this menu item.                       |
+| `\_invalid`               |                      Not mapped.                       |                     Whether the menu item represents an object that no longer exists.                     |
+
+-   [<a name="menu_item_menu_item_parent">1</a>] - the parent -> child relationship is expressed in block via the `innerBlocks` attribute and is therefore not required as a explicit block attribute.
+-   [<a name="menu_item_target">2</a>] - applies only if the value of the `target` field is `_blank`.
+
+### Inconsistencies
+
+#### Mapping
+
+For historical reasons, the following properties display some inconsistency in their mapping from Menu Item Object to Block attribute:
+
+-   `type` -> `kind` - the family of objects is stored as `kind` on the block and so must be mapped accordingly.
+-   `object` -> `type` - the type of object is stored as `type` on the block and so must be mapped accordingly.
+-   `object_id` -> `id` - the block stores a reference to the original object's ID as the `id` _attribute_. This should not be confused with the block's `clientId` which is unrelated.
+-   `attr_title` -> `title` - the HTML `title` attribute is stored as `title` on the block and so must be mapped accordingly.
+
+#### Object Types
+
+-   Menu Item objects which represent "Tags" are stored in WordPress as `post_tag` but the block expects their `type` attribute to be `tag` (omiting the `post_` suffix). This inconsistency is accounted for in [the mapping utilities methods](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/packages/edit-navigation/src/store/utils.js#L279-L281).
```

Comment:
As an aside, I think we can still change this to prefer `post_tag` before the navigation block is finalized. (We'd still need to account for older data that has `tag` though).